miscellaneous/validator-service/service.bal

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+
+import ballerina/http;
+import ballerina/log;
+import ballerinax/health.fhir.r4;
+import ballerinax/health.fhir.r4.validator;
+
+configurable string[] CORS_ALLOWED_ORIGINS = ?;
```

Comment:
Do we need to have the cors config in order the service to be consumed?

Reply:
Yes as this service is for react, cors config is compulsory.